.cirrus.yml

```diff
@@ -4,7 +4,6 @@ macos_instance:
 task:
   script: |
     brew install python@3.10
-    brew install python-tk@3.10
```

Comment:
@tylerjereddy - this shouldn't be necessary anymore (we don't import tkinter anywhere else), so I've gone ahead and removed it